### PR TITLE
MCA/COMMON/UCX: added synonym to opal_mem_hook variable

### PR DESCRIPTION
**Pull request by hoopoepg**
- added synonim to opal_mem_hook variable to allow
  to print it in opal_info -a

Signed-off-by: Sergey Oblomov <sergeyo@mellanox.com>

### CODE REVIEW

File: opal/mca/common/ucx/common_ucx.c [hunk 1]
```diff
@@ -34,6 +34,37 @@ static void opal_common_ucx_mem_release_cb(void *buf, size_t length,
     ucm_vm_munmap(buf, length);
 }
 
+OPAL_DECLSPEC void opal_common_ucx_mca_var_register(const mca_base_component_t *component)
+{
+    static int registered = 0;
+    static int hook_index;
+    if (!registered) {
+        mca_base_var_register("opal", "ucx", "*", "verbose",
```
Comment (jsquyres) [inline opal/mca/common/ucx/common_ucx.c]: @hoopoepg I brought up this MCA var on the weekly webex today.  People weren't wild about the idea of having a `*` in the var name -- among other reasons, it means you'll have to escape this var name on the shell command line.  Can you do something else, perhaps to effect an end name of `ucx_verbose` (with no middle part)?

Comment (hoopoepg) [reply opal/mca/common/ucx/common_ucx.c]: right, will fix var name

Comment (hoopoepg) [reply opal/mca/common/ucx/common_ucx.c]: @jsquyres fixed

**Comment by hoopoepg** [comment]
bot:mellanox:retest

**Comment by hoopoepg** [comment]
@jsquyres fixed variable name
@yosefe could you review?

File: opal/mca/common/ucx/common_ucx.c [hunk 2]
```diff
@@ -34,6 +34,37 @@ static void opal_common_ucx_mem_release_cb(void *buf, size_t length,
...
+{
+    static int registered = 0;
+    static int hook_index;
+    if (!registered) {
+        mca_base_var_register("opal", "opal_common", "ucx", "verbose",
+                              "Verbose level of the UCX components",
+                              MCA_BASE_VAR_TYPE_INT, NULL, 0, MCA_BASE_VAR_FLAG_SETTABLE,
+                              OPAL_INFO_LVL_3, MCA_BASE_VAR_SCOPE_LOCAL,
+                              &opal_common_ucx.verbose);
+        mca_base_var_register("opal", "opal_common", "ucx", "progress_iterations",
```
[the remark this answers is not in the document]
Comment (hoopoepg) [reply opal/mca/common/ucx/common_ucx.c]: added all variables

File: ompi/mca/pml/ucx/pml_ucx_component.c [hunk 3]
```diff
@@ -64,6 +64,7 @@ static int mca_pml_ucx_component_register(void)
                                            OPAL_INFO_LVL_3,
                                            MCA_BASE_VAR_SCOPE_LOCAL,
                                            &ompi_pml_ucx.num_disconnect);
+    opal_common_ucx_mca_var_register(&mca_pml_ucx_component.pmlm_version);
```
[the remark this answers is not in the document]
Comment (hoopoepg) [reply ompi/mca/pml/ucx/pml_ucx_component.c]: added to osc

File: oshmem/mca/spml/ucx/spml_ucx_component.c [hunk 4]
```diff
@@ -104,6 +104,8 @@ static int mca_spml_ucx_component_register(void)
                                     "Use non-blocking memory registration for shared heap",
                                     &mca_spml_ucx.heap_reg_nb);
 
+    opal_common_ucx_mca_var_register(&mca_spml_ucx_component.spmlm_version);
```
[the remark this answers is not in the document]
Comment (hoopoepg) [reply oshmem/mca/spml/ucx/spml_ucx_component.c]: added to atomic

**Comment by yosefe** [comment]
@hoopoepg pls pick to v4.0.x

**Pull request merged**